backend/vpnathome/apps/management/middleware.py

```diff
@@ -15,6 +15,9 @@ def get_init_sh_path():
     d = os.environ.get('VIRTUAL_ENV', '')
     return abspath(join(d, 'bin/init.sh')) if d else '${VIRTUAL_ENV}/bin/init.sh'
 
```

Comment:
There should be double line separator here.

This stuff is normally checked by linter following https://www.python.org/dev/peps/pep-0008/#blank-lines
rule, but since I use PyCharm, pep8 linter is built-in, I never bothered to integrate linter with `Makefile`.

I guess this doesn't work for everybody, so if you want, I'd be glad to add pep8 linter to the build system.

If you don't feel like it's your cup of tea, I'll merge it as it is and do linting separately.

Let me know if you want to follow-up.

Reply:
I do use a PEP-8 linting package on Atom but I didn't think to fix that or anything because that line separating issue wasn't really relevant with my pull request.

I'll fix it though. I'll just add another commit and add the separations.

Reply:
Oops, sorry about that. I do have a linter on Atom but I didn't realise about the line separators issue. I'll just add another commit and fix that.